src/test/java/com/google/api/codegen/testdata/php_test_library.baseline

```diff
@@ -442,6 +524,41 @@ class LibraryServiceTest extends TestCase
     /**
      * @test
      */
+    public function listStringsTest()
+    {
+        list($grpcStub, $client) = $this->createStubAndClient([$this, 'createMockLibraryServiceImpl'], 'createLibraryServiceStubFunction');
+
+        $this->assertTrue($grpcStub->isExhausted());
+
+        // Mock response
+        $nextPageToken = "";
+        $stringsElement = "stringsElement474465855";
+        $strings = [$stringsElement];
+        $expectedResponse = new ListStringsResponse();
+        $expectedResponse->setNextPageToken($nextPageToken);
+        foreach ($strings as $elem) {
+            $expectedResponse->addStrings($elem);
+        }
+        $grpcStub->addResponse($expectedResponse);
+
+        $response = $client->listStrings();
+        $actualRequests = $grpcStub->getReceivedCalls();
+        $this->assertSame(1, count($actualRequests));
```

Comment:
You use assertEquals(...) in L553 and assertSame(...) here. What is the difference?